addons/sys_data/fnc_getPlayerRadioList.sqf

```diff
@@ -22,9 +22,10 @@ if (alive acre_player) then {
     private _weapons = [acre_player] call EFUNC(sys_core,getGear);
     _radioList = _weapons select {_x call EFUNC(sys_radio,isUniqueRadio)};
 
-    // Remove those radios that are being actively used by other players.
-    private _externalRadiosInUse = _radioList select {_x call EFUNC(sys_external,isExternalRadioUsed)};
-    _radioList = _radioList - _externalRadiosInUse;
+    // Remove those radios that are being actively used by other players
+    _radioList = _radioList select {!(_x call EFUNC(sys_external,isExternalRadioUsed))};
```

Comment:
This could be merged with L23 and form:
```
_radioList = _weapons select {_x call EFUNC(sys_radio,isUniqueRadio) && {!(_x call EFUNC(sys_external,isExternalRadioUsed))}};
```
for faster iteration (from 2n to n).